### DISCUSSION TITLE
setup question, multible usb soundcards.

### DISCUSSION THREAD

**Discussion by foxy-1423** [VoidXH/Cavern#117] [General]
so im wondering if caven supports multiple independent sound cards.
so the idea is to constuct a super budget 9.2.4 surround setup using a bunch of cheapo used speakers and amps together with a bunch of cheap usb sound cards. then have cavern decode the atoms movie and send the decoded signals to each dac, so like the front left and right would be handled by its own card the surrounds would have their own cards and so-on. 

is this someting cavern supports? esentially sending the decoded audio to specific outputs on a PC?

**Comment by VoidXH** [comment]
Yes, you can use ASIO4ALL's aggregate sound card option, but you need a player which supports very high channel counts, and that is the tricky part. According to the guys at QuadraphonicQuad, JRiver works, but only when there's no video.

**Reply by VoidXH** [reply]
I don't know of any. This is the expertise of forums.

**Comment by TheTrueCoder** [comment]
VB Audio make programs called Matrix and Coconut which are two different sized routing matrices to/from Windows audio devices, ASIO, VBAN network audio, and emulated virtual audio devices that you can play to in windows. Hope this helps, it's a really powerful and intuitive piece of software. https://vb-audio.com/Matrix/index.htm
